### PR TITLE
Cleanup integration tests.

### PR DESCRIPTION
**Pull request by olupton**
**Description**
The `ring_different_seed` test was not actually enabled, and in any case the ringtest is not sensitive to the Random123 seed.

### CODE REVIEW

**Comment by olupton** [comment]
The failing test passed when I retried it.

File: tests/integration/CMakeLists.txt [hunk 1]
```diff
@@ -61,46 +61,36 @@ endif()
 # As reports require MPI, do not add test if report is enabled.
 # GPU tests need SLURM allocation and hence they won't run without srun
```
Comment (iomaganaris) [inline tests/integration/CMakeLists.txt:62]: Nitpick.. Comment not relevant any more?

**Review by iomaganaris — APPROVED** [review]
LGTM :+1:

**Review by pramodk — APPROVED** [review]
LGTM

> for future reference, test related to global seed index should be covered by something like: https://github.com/neuronsimulator/testcorenrn/pull/11

**Pull request merged**